css/types/time-percentage.json

```diff
@@ -0,0 +1,58 @@
+{
+  "css": {
+    "types": {
+      "time-percentage": {
+        "__compat": {
+          "description": "&lt;time-percentage&gt;",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/time-percentage",
+          "support": {
+            "chrome": {
+              "version_added": true
```

Comment:
Have you tried this? I can't find any code evidence of it.

Reply:
time is used for things like animation eg: https://developer.mozilla.org/en-US/docs/Web/CSS/animation-duration

This is just a data type that would allow a time or a percentage. So if time is supported and percentage is supported ... it's supported.

Reply:
@rachelandrew have you had a chance to look into this?

Reply:
I need confirmation on this. If you know of a version of Chrome where this works, then the current version has a bug and I need to open a ticket.

Reply:
I'm not really sure what the issue is to be honest. This is a data type, it may or may not be supported for a particular property, but that would be an issue to raise against the page/compat data for that property don't you think?

And sorry for delay in replying I didn't see the original notification and hadn't circled back to check on this.